### PR TITLE
what did the Doppelganger say? IMU

### PR DESCRIPTION
**Pull request by DiegoLHendrix**

### CODE REVIEW

**Review by pro-sper — APPROVED** [review]
It looks correct to me

File: include/IMU.hpp [hunk 1]
```diff
@@ -74,341 +83,59 @@ class IMU {
...
-            .Type = nullptr,
-            .Data = (uintptr_t) &vectorZValues[2],
-        },
-        {
-            .Key = CO_KEY(0x2102, 3, CO_UNSIGNED16 | CO_OBJ___PRW),
-            .Type = nullptr,
-            .Data = (uintptr_t) &vectorZValues[3],
-        },
-        CO_OBJ_DIR_ENDMARK,
+        MANDATORY_IDENTIFICATION_ENTRIES_1000_1014,
```
Comment (mjmagee991) [inline include/IMU.hpp:96]: Add comments to the dictionary similar to the examples

File: include/IMU.hpp [hunk 2]
```diff
@@ -74,341 +83,59 @@ class IMU {
...
+        TRANSMIT_PDO_MAPPING_ENTRY_1AXX(0x01, 3, PDO_MAPPING_UNSIGNED16),
+        TRANSMIT_PDO_MAPPING_ENTRY_1AXX(0x01, 4, PDO_MAPPING_UNSIGNED16),
+
+        TRANSMIT_PDO_MAPPING_START_KEY_1AXX(0x02, 0x04),
+        TRANSMIT_PDO_MAPPING_ENTRY_1AXX(0x02, 1, PDO_MAPPING_UNSIGNED16),
+        TRANSMIT_PDO_MAPPING_ENTRY_1AXX(0x02, 2, PDO_MAPPING_UNSIGNED16),
+        TRANSMIT_PDO_MAPPING_ENTRY_1AXX(0x02, 3, PDO_MAPPING_UNSIGNED16),
+        TRANSMIT_PDO_MAPPING_ENTRY_1AXX(0x02, 4, PDO_MAPPING_UNSIGNED16),
+
+        DATA_LINK_START_KEY_21XX(0x00, 0x04),
```
Comment (mjmagee991) [inline include/IMU.hpp:125]: @ActuallyTaylor Are these data links set up right?

File: targets/DEV1-IMU/main.cpp [hunk 3]
```diff
@@ -36,62 +34,40 @@ namespace DEV = EVT::core::DEV;
...
+IO::UART& uart = IO::getUART<IO::Pin::UART_TX, IO::Pin::UART_RX>(9600);
+
 // create a can interrupt handler
 void canInterrupt(IO::CANMessage& message, void* priv) {
-    EVT::core::types::FixedQueue<CANOPEN_QUEUE_SIZE, IO::CANMessage>* queue =
-        (EVT::core::types::FixedQueue<CANOPEN_QUEUE_SIZE, IO::CANMessage>*) priv;
-
-    // Log raw received data
-    log::LOGGER.log(log::Logger::LogLevel::DEBUG, "Got RAW message from %X of length %d with data: ", message.getId(), message.getDataLength());
+    auto* queue = (EVT::core::types::FixedQueue<CANOPEN_QUEUE_SIZE, IO::CANMessage>*) priv;
```
Comment (mjmagee991) [inline targets/DEV1-IMU/main.cpp:39]: Change this function back to be the way it was initially

File: targets/DEV1-IMU/main.cpp [hunk 4]
```diff
@@ -36,62 +34,40 @@ namespace DEV = EVT::core::DEV;
 * @param message[in] The passed in CAN message that was read.
 */
 
+IO::UART& uart = IO::getUART<IO::Pin::UART_TX, IO::Pin::UART_RX>(9600);
```
Comment (mjmagee991) [inline targets/DEV1-IMU/main.cpp]: Don't use a global UART instance. Use the logger

File: targets/DEV1-IMU/main.cpp [hunk 5]
```diff
@@ -36,62 +34,40 @@ namespace DEV = EVT::core::DEV;
...
-extern "C" void COPdoTransmit(CO_IF_FRM* frm) {}
-
-extern "C" int16_t COPdoReceive(CO_IF_FRM* frm) { return 0; }
-
-extern "C" void COPdoSyncUpdate(CO_RPDO* pdo) {}
-
-extern "C" void COTmrLock(void) {}
-
-extern "C" void COTmrUnlock(void) {}
+//setup a TPDO event handler to print the raw TPDO message when sending
```
Comment (mjmagee991) [inline targets/DEV1-IMU/main.cpp]: Delete this function. It's not necessary for main code

**Review by mjmagee991 — CHANGES_REQUESTED** [review]
Some cleanup to do

**Review by mjmagee991 — APPROVED** [review]
Looks great

**Pull request merged**